starfish/intensity_table.py

```diff
@@ -276,3 +276,14 @@ def synthetic_intensities(
         intensities[cls.Constants.GENE.value] = ('features', genes)
 
         return intensities
+
+    def mask_low_intensity_features(self, intensity_threshold):
```

Comment:
this is another potential source of variation between MERFISH results on this branch and master. the MERFISH decoder in master discards pixel vectors with an _L2 norm_ lower than a specified threshold: 

```pixel_traces, pixel_traces_l2_norm = self._parse_pixel_traces(encoded)
        local_magnitude = np.reshape(pixel_traces_l2_norm, img_size)
        decoded_img[local_magnitude < self.magnitude_threshold] = 0
```

whereas the code in this branch filters out pixel vectors below a specified _mean intensity_

Reply:
i do, however like the masking approach (to ignore these vectors) rather than the set things to zero and do all the nearest neighbor calculations anyways.

Reply:
Unfortunately as coded it does the nearest neighbor calculations so there is no efficiency gain... but I bet I might be able to pre-mask the way you suggest.